CHANGELOG.rst

```diff
@@ -2,8 +2,9 @@
 CHANGELOG
 =========
 
```

Comment:
I think we should also leave an empty “Unreleased” section for future devs to add changes there alongside their PRs.